packages/spark-core/base/_tables.scss

```diff
@@ -7,32 +7,57 @@
   overflow: auto;
 }
 
-.sprk-b-Table__head,
-.sprk-b-Table__row-heading {
-  background-color: $blue-dark;
+.sprk-b-Table tr {
+  border-bottom: 1px solid $gray-light;
+}
+
+.sprk-b-Table th {
+  background-color: $black;
```

Comment:
Hey @bobwise  don't forgot to make everything that could be "theme-able" into vars.